exercises/I-floats/exercise.js

```diff
@@ -1,2 +1,14 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+
+var studentsPreciseRatio = (15/(15+8))*100;
```

Comment:
It looks like you're initialising variables above, but you're not actually using them here. Imagine if we wanted to change the value of the numberOfStudents variable - if we were referring to those variables in our calculations, we wouldn't need to change anything else in the code.